src/pages/backstop.tsx

```diff
@@ -63,18 +63,20 @@ const Backstop: NextPage = () => {
   };
 
   async function getLPEstimate(amount: bigint, depositTokenAddress: string) {
-    const estimate =
-      (await backstopMintByDepositTokenAmount(
-        {
-          depositTokenAddress: depositTokenAddress,
-          depositTokenAmount: amount,
-          minLPTokenAmount: BigInt(0),
-          user: walletAddress,
-        },
-        true,
-        backstopData?.config.backstopTkn || ''
-      )) || BigInt(0);
-    return estimate;
+    let response = await backstopMintByDepositTokenAmount(
+      {
+        depositTokenAddress: depositTokenAddress,
+        depositTokenAmount: amount,
+        minLPTokenAmount: BigInt(0),
+        user: walletAddress,
+      },
+      true,
+      backstopData?.config.backstopTkn || ''
+    );
+    if (response) {
+      return response.result.isOk() ? response.result.unwrap() : BigInt(0);
```

Comment:
I think the wallet function is better off returning the ContractResponse as it doesn't want to assume how the response will be used, or we should better separate our simulation and submission code to avoid any accidental bugs by using things incorrectly.

Reply:
i see , that makes sense just a little weird to have to use .isOk() ? response.result.unwrap() every time you need the value